src/com/mendix/widget/carousel/components/__tests__/Carousel.spec.ts

```diff
@@ -0,0 +1,113 @@
+import { ReactWrapper, ShallowWrapper, mount, shallow } from "enzyme";
+import { DOM, createElement } from "react";
+
+import { Carousel, CarouselProps, Image } from "../Carousel";
+import { CarouselItem } from "../CarouselItem";
+
+describe("Carousel", () => {
+    let images: Image[];
+    describe("shallow component", () => {
```

Comment:
Decided to separate the shallow and full mount tests. Seems neater. Otherwise I couldn't avoid breaking at least one previous review suggestion :D